inference-engine/src/mkldnn_plugin/nodes/mkldnn_activation_node.cpp

```diff
@@ -111,6 +111,18 @@ caseless_map<std::string, std::function<void(GenericLayer*, mkldnn::algorithm&,
             beta = 0.0f;
             algorithm = eltwise_hsigmoid;
         }},
+        {"round", [](GenericLayer* activationLayer, mkldnn::algorithm& algorithm, float& alpha, float& beta) {
+            alpha = 0.0f;
+            beta = 0.0f;
+
+            std::string mode = activationLayer->GetParamAsString("mode", "half_to_even");
+            if (mode == "half_to_even")
+                algorithm = eltwise_round_half_to_even;
+            else if (mode == "half_away_from_zero")
+                algorithm = eltwise_round_half_away_from_zero;
+            else
+                THROW_IE_EXCEPTION << "Incorrect mode for round operation.";
```

Comment:
I would say this is not correct message. Example: Round operation in Ngraph was extended to support one more mode, while still doesn't support it. Better message would be "Round layer with name "name" doesn't support mode "mode""

Reply:
Thank you, I changed it